free-programming-books.md

```diff
@@ -1402,6 +1410,8 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Modern Java EE Design Patterns](http://www.oreilly.com/programming/free/modern-java-ee-design-patterns.csp) - Markus Eisele (email address *requested*, not required)
 * [Neural Network Development with Java](https://www.packtpub.com/packt/free-ebook/neural-networks-java) - Alan M. F. Souza and Fabio M. Soares (email address *requested*, not required)
 * [Object-Oriented vs. Functional Programming](http://www.oreilly.com/programming/free/object-oriented-vs-functional-programming.csp) - Richard Warburton (email address *requested*, not required)
+* [Object-oriented Programming in JavaTM Textbook](http://computing.southern.edu/halterman/OOPJ/) - Rick Halterman (PDF per Chapter)
+* [Object Oriented Programming using Java](https://bookboon.com/en/object-oriented-programming-using-java-ebook) - Simon Kendal (PDF)
```

Comment:
Alphabetization: "Object " < "Object-"

Reply:
LOL seems tough. hope there's an automa that can give us all that error indications

Reply:
There is. It's Travis. Click on "details"

> continuous-integration/travis-ci/pr — The Travis CI build failed              Details

Reply:
> There is. It's Travis. Click on "details"
> 
> > continuous-integration/travis-ci/pr — The Travis CI build failed              Details
I can't see a details  button where is it? 
in that page somewhere between our conversations i see
```
harrykar and others added some commits 8 days ago
harrykar
issue #3011 Java, Java, Java Object-Oriented Problem Solving by R. Mo…
e10717d
@eshellman
harrykar
Revert 3021 master (#3025)
f7d7261
harrykar
issues:add in list #3009,#3010, correction blank in #3011
227a313
harrykar
issue #3011 Java, Java, Java Object-Oriented Problem Solving by R. Mo…
45ad31c
harrykar
add 2 books, create [Eiffel](#eiffel),
898d962
harrykar
correct merge annotations floated around
166fb2a
harrykar
correct add one blankline before ### Subversion
1d291eb

```
the last part is the SHA1 and is a link you mean that one?

Reply:
Don't mind I found it :)